### ISSUE TITLE
Still issues with bracket imports

### ISSUE THREAD

**Issue by Sidneys1** [austinbenincasa/Abos#1]
Had to make a small change to get it to build (under bash-on-windows):
```diff
diff --git a/libc/include/string.h b/libc/include/string.h
index 5216826..f5cd579 100644
--- a/libc/include/string.h
+++ b/libc/include/string.h
@@ -1,7 +1,7 @@
 #ifndef _STRING_H
 #define _STRING_H 1

-#include <sys/cdefs.h>
+#include "sys/cdefs.h"

 #include <stddef.h>
```

Eventually worked though!
![qemu-system-i386_2017-11-10_14-42-53](https://user-images.githubusercontent.com/1771233/32675626-795cd78c-c625-11e7-8604-b02c8f1e770c.png)

**Comment by austinbenincasa** [comment]
@Sidneys1 Weird, I didn't have this issue on Ubuntu. Fixed this issue in commit 84b00b62f798652dec3d3eb6979baab9702ede2e

**Comment by austinbenincasa** [comment]
Should all be fixed now with the merging of your pull request
